client/.env

```diff
@@ -1 +1,3 @@
-VITE_API_URL=http://localhost:8000
\ No newline at end of file
+VITE_API_URL=http://localhost:8000/api
+# uncomment this to pull PDFs and DI results from local storage
+# STORAGE_TYPE=LOCAL
```

Comment:
Great points, the local experience will stay the default for now. Updated the `STORAGE_TYPE` variable to `VITE_DOCUMENT_BLOB_STORAGE_GENERATE_SAS_URL`. We will connect with you soon about this, but we needed a way to get the di results and pdf from a private storage account. The function backend generates a SAS URL for the pdf and di results and sends it back to the client so it can pull it up. The default experience is just using the urls that comes from the backend store, so nothing should change there